pkg/rtc/subscribedtrack.go

```diff
@@ -227,8 +227,8 @@ func (t *SubscribedTrack) SetRTPSender(sender *webrtc.RTPSender) {
 }
 
 func (t *SubscribedTrack) updateDownTrackMute() {
-	muted := t.subMuted.Load() || t.pubMuted.Load()
-	t.DownTrack().Mute(muted)
+	t.DownTrack().Mute(t.subMuted.Load())
+	t.DownTrack().PubMute(t.pubMuted.Load())
```

Comment:
Splitting the subscriber mute vs publisher mute. `sfu.DownTrack` will handle these differently.